prometheus/common/alert-rules.d/crunchy-alert-rules-patroni.yml.example

```diff
@@ -0,0 +1,73 @@
+###
+#
+# Copyright © 2017-2022 Crunchy Data Solutions, Inc. All Rights Reserved.
+#
+###
+
+groups:
+- name: alert-rules
+  rules:
+
+########## PATRONI RULES ##########
+
+  - alert: PatroniPGIsRunning
+    expr: patroni_postgres_running != 1
+    for: 60s
+    labels:
+      service: patroni 
+      severity: critical
+      severity_num: 300
+    annotations:
+      summary: 'Patroni is reporting that PostgreSQL is not running on {{ $labels.job }}'
+
+  - alert: PatroniPendingRestart
+    expr: patroni_pending_restart != 0
+    for: 60s
+    labels:
+      service: patroni 
+      severity: critical
```

Comment:
not warning ?
or at least give it more time at line 25

Reply:
Increasing this shouldn't be a problem to give it more time to self resolve. However, the timer at line 25 is the PENDING alert state. It does not send out anything at all, warning or critical alert, for this time.

Reply:
Changed this to warning and Pending state to 30 minutes to allow time for a playbook run to complete before possibly triggering false alarms